app/controllers/api_controller.rb

```diff
@@ -44,6 +44,7 @@ def current_sync_region
   end
 
   def block_level_sync?
+    return false unless current_user
```

Comment:
You could tack on a `current_user &&` on the next line if you want 😅 

I'll let you pick whichever seems more readable/natural 👍

Reply:
I actually switched from `current_user&.` to `current_user &&` and then to this 😛 . I tried to state this explicitly, in a similar style as the `current_sync_region` method.